docs/examples/generatedCanonicalFields.json

```diff
@@ -0,0 +1,822 @@
+    "shortCode": "Org Legal Docs",
+    "dataType": "string"
+  },
+  {
+    "label": "Org Board Members",
+    "shortCode": "Org Board Members",
+    "dataType": "string"
+  },
+  {
+    "label": "Org Administrative Lead",
+    "shortCode": "Org Administrative Lead",
+    "dataType": "string"
+  },
+  {
+    "label": "Org Annual Operating Budget",
+    "shortCode": "Org Annual Operating Budget",
+    "dataType": "number"
+  },
+  {
+    "label": "Org Lobbying Activities",
+    "shortCode": "Org Lobbying Activities",
+    "dataType": "string"
+  },
+  {
+    "label": "Org Start Date",
+    "shortCode": "Org Start Date",
+    "dataType": "string"
+  },
+  {
+    "label": "Org Geographic Areas",
+    "shortCode": "Org Geographic Areas",
+    "dataType": "string"
+  },
+  {
+    "label": "Org Audited Financial Statements",
+    "shortCode": "Org Audited Financial Statements",
+    "dataType": "string"
+  },
+  {
+    "label": "Org Grant Agreement Signatory",
+    "shortCode": "Org Grant Agreement Signatory",
+    "dataType": "string"
+  },
+  {
+    "label": "Org Fiscal Year End Date",
+    "shortCode": "Org Fiscal Year End Date",
+    "dataType": "string"
+  },
+  {
+    "label": "Proposal Name(s)",
+    "shortCode": "Proposal Name(s)",
+    "dataType": "string"
+  },
+  {
+    "label": "Proposal Primary Contact Name",
+    "shortCode": "Proposal Primary Contact Name",
+    "dataType": "string"
+  },
+  {
+    "label": "Proposal Primary Email",
+    "shortCode": "Proposal Primary Email",
+    "dataType": "string"
+  },
+  {
+    "label": "Proposal Primary Phone Number",
+    "shortCode": "Proposal Primary Phone Number",
+    "dataType": "string"
+  },
+  {
+    "label": "Proposal Summary",
+    "shortCode": "Proposal Summary",
+    "dataType": "string"
+  },
+  {
+    "label": "Proposal Budget",
+    "shortCode": "Proposal Budget",
+    "dataType": "string"
+  },
+  {
+    "label": "Proposal Investment Start Date/Duration",
+    "shortCode": "Proposal Investment Start Date/Duration",
+    "dataType": "string"
+  },
+  {
+    "label": "Proposal Investment End Date",
+    "shortCode": "Proposal Investment End Date",
+    "dataType": "string"
+  },
+  {
+    "label": "Proposal Project Total Budget",
+    "shortCode": "Proposal Project Total Budget",
+    "dataType": "number"
+  },
+  {
+    "label": "Proposal Type",
+    "shortCode": "Proposal Type",
+    "dataType": "string"
+  },
+  {
+    "label": "Proposal Fiscal Sponsorship Org Name",
+    "shortCode": "Proposal Fiscal Sponsorship Org Name",
+    "dataType": "string"
+  },
+  {
+    "label": "Proposal Location of Work (free-form string)",
+    "shortCode": "Proposal Location of Work (free-form string)",
+    "dataType": "string"
+  },
+  {
+    "label": "Proposal Involves Lobbying or Electioneering?",
+    "shortCode": "Proposal Involves Lobbying or Electioneering?",
+    "dataType": "string"
+  },
+  {
+    "label": "IRB review needed?",
+    "shortCode": "IRB review needed?",
```

Comment:
More potential style guide ideas:

1. Name boolean fields the way we would name a variable (e.g. `isBlah` or `hasBlah` or `needsBlah`)
2. Don't include acronyms in short codes

So this shortCode would be `needsInternalReviewBoard`.

Reply:
Suppose a new column were created in the original spreadsheet(s) that includes a PDC shortCode? Then the translation from PDC canonicalField to shortCode would be clearer end-to-end. What do you think?